src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -122,37 +127,6 @@ def get_original_timestamps(self, stub_test: bool = False) -> list[np.ndarray]:
                 timestamps.append(video.get_video_timestamps(max_frames=max_frames))
         return timestamps
 
-    def get_timing_type(self) -> Literal["starting_time and rate", "timestamps"]:
```

Comment:
This should also be removed for InternalVideoInterface right?

Reply:
Yes.

Reply:
Should be removed.